src/App.jsx

```diff
@@ -1,9 +1,19 @@
-import { HomePage } from './pages';
+import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
+import { Header } from './components/organisms/Header/Header';
+import { HomePage, ProfilePage, EditProfilePage, ChatPage } from './pages';
 
 export const App = () => {
     return (
-        <main>
-            <HomePage />
-        </main>
+        <Router>
+            <Header/>
+            <main>
+                <Routes>
+                    <Route path="/" exact element={<HomePage/>} />
+                    <Route path="/profile"  element={<ProfilePage/>} />
```

Comment:
Profile page should have the :id parameter (somehow we need to know which data we should fetch)

Reply:
Done